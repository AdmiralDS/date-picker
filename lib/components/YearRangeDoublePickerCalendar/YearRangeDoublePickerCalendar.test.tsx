import { render } from '@testing-library/react';
import { ThemeProvider } from 'styled-components';

import { LIGHT_THEME } from '@admiral-ds/react-ui';
import { YearRangeDoublePickerCalendar } from '@admiral-ds/date-picker';

describe('YearRangeDoublePickerCalendar', () => {
  test('should render component', () => {
    const { container } = render(
      <ThemeProvider theme={LIGHT_THEME}>
        <YearRangeDoublePickerCalendar />
      </ThemeProvider>,
    );
    expect(container).toMatchSnapshot();
  });
});
