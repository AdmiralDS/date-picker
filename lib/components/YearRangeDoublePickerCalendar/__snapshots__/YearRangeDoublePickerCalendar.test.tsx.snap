// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`YearRangeDoublePickerCalendar > should render component 1`] = `
<div>
  <div
    class="calendarStyle__DoublePickerCalendarWrapper-sc-1gbkw52-1 jXswWu"
  >
    <div
      class="calendarStyle__SingleContainer-sc-1gbkw52-2 fGmozL"
    >
      <div
        class="TwentyYearsNavigationPanelWidget__TwentyYearsNavigationPanelWrapper-sc-f0rhxx-0 jPPWCo"
      >
        <button
          class="sc-dCVDEO kvSYsL"
          data-panel-target-type="left"
          type="button"
        >
          <div
            aria-hidden="true"
            class="sc-kiMgGE hzqvwk"
          />
          <div
            aria-hidden="true"
            class="sc-cNFqVt dRewJq"
          >
            <svg
              fill="none"
              focusable="false"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M14.5 18.75c.18 0 .35-.07.48-.21.24-.27.22-.68-.04-.92l-6.37-5.74 6.41-5.55a.65.65 0 0 0 .04-.92.66.66 0 0 0-.92-.04l-6.9 5.99c-.31.28-.31.76 0 1.04l6.86 6.18c.13.12.28.17.44.17"
                fill="#74767B"
              />
            </svg>
          </div>
        </button>
        <div
          class="TwentyYearsNavigationPanelWidget__TwentyYearsWrapper-sc-f0rhxx-1 jNsNII"
        >
          2021â€“2040
        </div>
        <button
          class="sc-dCVDEO kvSYsL"
          data-panel-target-type="right"
          type="button"
        >
          <div
            aria-hidden="true"
            class="sc-kiMgGE hzqvwk"
          />
          <div
            aria-hidden="true"
            class="sc-cNFqVt dRewJq"
          >
            <svg
              fill="none"
              focusable="false"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M9.47 18.75c-.18 0-.35-.07-.48-.21a.66.66 0 0 1 .04-.92l6.37-5.74-6.41-5.55a.65.65 0 0 1-.04-.92c.24-.26.65-.28.92-.04l6.9 5.99c.31.28.31.76 0 1.04l-6.86 6.18c-.13.12-.28.17-.44.17"
                fill="#74767B"
              />
            </svg>
          </div>
        </button>
      </div>
      <div
        class="calendarStyle__CalendarContainer-sc-1gbkw52-3 iTszZj"
      >
        <div
          class="YearsOfTwentyYearsWidget__YearsOfYearWrapper-sc-o46cop-0 kXSecx calendarStyle__YearRangeCalendarView-sc-1gbkw52-6 hHnqcQ"
          data-container-type="yearsOfYearWrapper"
        >
          <div
            class="Years__YearsWrapper-sc-4dphf0-0 gsWMmH"
            data-container-type="yearsWrapper"
          >
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-is-start-of-week-cell="true"
              data-value="Fri, 01 Jan 2021 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 gNZoYi"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2021
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-value="Sat, 01 Jan 2022 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2022
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-value="Sun, 01 Jan 2023 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2023
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-is-end-of-week-cell="true"
              data-value="Mon, 01 Jan 2024 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 dYbKaZ"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2024
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-is-current-day-cell="true"
              data-is-start-of-week-cell="true"
              data-value="Wed, 01 Jan 2025 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 gNZoYi"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 fbZjLX"
              >
                2025
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-value="Thu, 01 Jan 2026 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2026
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-value="Fri, 01 Jan 2027 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2027
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-is-end-of-week-cell="true"
              data-value="Sat, 01 Jan 2028 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 dYbKaZ"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2028
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-is-start-of-week-cell="true"
              data-value="Mon, 01 Jan 2029 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 gNZoYi"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2029
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-value="Tue, 01 Jan 2030 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2030
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-value="Wed, 01 Jan 2031 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2031
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-is-end-of-week-cell="true"
              data-value="Thu, 01 Jan 2032 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 dYbKaZ"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2032
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-is-start-of-week-cell="true"
              data-value="Sat, 01 Jan 2033 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 gNZoYi"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2033
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-value="Sun, 01 Jan 2034 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2034
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-value="Mon, 01 Jan 2035 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2035
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-is-end-of-week-cell="true"
              data-value="Tue, 01 Jan 2036 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 dYbKaZ"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2036
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-is-start-of-week-cell="true"
              data-value="Thu, 01 Jan 2037 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 gNZoYi"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2037
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-value="Fri, 01 Jan 2038 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2038
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-value="Sat, 01 Jan 2039 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2039
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-is-end-of-week-cell="true"
              data-value="Sun, 01 Jan 2040 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 dYbKaZ"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2040
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="calendarStyle__SingleContainer-sc-1gbkw52-2 fGmozL"
    >
      <div
        class="TwentyYearsNavigationPanelWidget__TwentyYearsNavigationPanelWrapper-sc-f0rhxx-0 jPPWCo"
      >
        <button
          class="sc-dCVDEO kvSYsL"
          data-panel-target-type="left"
          type="button"
        >
          <div
            aria-hidden="true"
            class="sc-kiMgGE hzqvwk"
          />
          <div
            aria-hidden="true"
            class="sc-cNFqVt dRewJq"
          >
            <svg
              fill="none"
              focusable="false"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M14.5 18.75c.18 0 .35-.07.48-.21.24-.27.22-.68-.04-.92l-6.37-5.74 6.41-5.55a.65.65 0 0 0 .04-.92.66.66 0 0 0-.92-.04l-6.9 5.99c-.31.28-.31.76 0 1.04l6.86 6.18c.13.12.28.17.44.17"
                fill="#74767B"
              />
            </svg>
          </div>
        </button>
        <div
          class="TwentyYearsNavigationPanelWidget__TwentyYearsWrapper-sc-f0rhxx-1 jNsNII"
        >
          2041â€“2060
        </div>
        <button
          class="sc-dCVDEO kvSYsL"
          data-panel-target-type="right"
          type="button"
        >
          <div
            aria-hidden="true"
            class="sc-kiMgGE hzqvwk"
          />
          <div
            aria-hidden="true"
            class="sc-cNFqVt dRewJq"
          >
            <svg
              fill="none"
              focusable="false"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M9.47 18.75c-.18 0-.35-.07-.48-.21a.66.66 0 0 1 .04-.92l6.37-5.74-6.41-5.55a.65.65 0 0 1-.04-.92c.24-.26.65-.28.92-.04l6.9 5.99c.31.28.31.76 0 1.04l-6.86 6.18c-.13.12-.28.17-.44.17"
                fill="#74767B"
              />
            </svg>
          </div>
        </button>
      </div>
      <div
        class="calendarStyle__CalendarContainer-sc-1gbkw52-3 iTszZj"
      >
        <div
          class="YearsOfTwentyYearsWidget__YearsOfYearWrapper-sc-o46cop-0 kXSecx calendarStyle__YearRangeCalendarView-sc-1gbkw52-6 hHnqcQ"
          data-container-type="yearsOfYearWrapper"
        >
          <div
            class="Years__YearsWrapper-sc-4dphf0-0 gsWMmH"
            data-container-type="yearsWrapper"
          >
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-is-start-of-week-cell="true"
              data-value="Tue, 01 Jan 2041 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 gNZoYi"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2041
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-value="Wed, 01 Jan 2042 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2042
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-value="Thu, 01 Jan 2043 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2043
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-is-end-of-week-cell="true"
              data-value="Fri, 01 Jan 2044 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 dYbKaZ"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2044
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-is-start-of-week-cell="true"
              data-value="Sun, 01 Jan 2045 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 gNZoYi"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2045
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-value="Mon, 01 Jan 2046 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2046
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-value="Tue, 01 Jan 2047 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2047
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-is-end-of-week-cell="true"
              data-value="Wed, 01 Jan 2048 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 dYbKaZ"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2048
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-is-start-of-week-cell="true"
              data-value="Fri, 01 Jan 2049 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 gNZoYi"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2049
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-value="Sat, 01 Jan 2050 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2050
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-value="Sun, 01 Jan 2051 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2051
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-is-end-of-week-cell="true"
              data-value="Mon, 01 Jan 2052 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 dYbKaZ"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2052
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-is-start-of-week-cell="true"
              data-value="Wed, 01 Jan 2053 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 gNZoYi"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2053
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-value="Thu, 01 Jan 2054 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2054
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-value="Fri, 01 Jan 2055 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2055
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-is-end-of-week-cell="true"
              data-value="Sat, 01 Jan 2056 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 dYbKaZ"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2056
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-is-start-of-week-cell="true"
              data-value="Mon, 01 Jan 2057 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 gNZoYi"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2057
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-value="Tue, 01 Jan 2058 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2058
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-value="Wed, 01 Jan 2059 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 jPDyjj"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2059
              </div>
            </div>
            <div
              class="DefaultCell__CellContainer-sc-e57mvj-0 iTTpCB"
              data-cell-type="yearCell"
              data-is-end-of-week-cell="true"
              data-value="Thu, 01 Jan 2060 09:00:00 GMT"
            >
              <div
                class="DefaultCell__LeftHalf-sc-e57mvj-1 HEkgM"
              />
              <div
                class="DefaultCell__RightHalf-sc-e57mvj-2 dYbKaZ"
              />
              <div
                class="DefaultCell__Cell-sc-e57mvj-3 cTQWMA"
              >
                2060
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
